erpnext/regional/india/setup.py

```diff
@@ -625,23 +636,6 @@ def get_custom_fields():
 				'fieldtype': 'Data',
 				'insert_after': 'pan',
 				'depends_on': 'eval:doc.is_transporter'
-			},
-			{
-				'fieldname': 'gst_category',
```

Comment:
This should be moved to Address, otherwise newer installations won't have these fields

Reply:
[Its added here](https://github.com/frappe/erpnext/pull/27885/files/378d30f75ba23451c0500a577297dd09ea5b2787#diff-4e7e9e05f08b10c2e67ff4bae8bdb93b942d84414b86cde3f11b46065eba7d30R488)